cvr/src/main/java/org/cbioportal/cmo/pipelines/cvr/mutation/CVRMutationDataReader.java

```diff
@@ -175,16 +175,16 @@ public void handleLine(String line) {
         }
         reader.close();
         log.info("Loaded " + String.valueOf(recordsToAnnotate.size()) + " records from MAF");
-        annotateRecordsWithPOST(recordsToAnnotate);
+        annotateRecordsWithPOST(recordsToAnnotate, forceAnnotation);
     }
 
-    private void annotateRecordsWithPOST(List<MutationRecord> records) throws Exception {
+    private void annotateRecordsWithPOST(List<MutationRecord> records, boolean reannotate) throws Exception {
```

Comment:
Explained above but just to clarify - at some point we lost the use of the force annotation flag. We are putting it back in. But we also need to make sure we can still differentiate between records read from JSON and records read from MAF.